Makefile

```diff
@@ -22,6 +22,8 @@ setup-self:
 	python setup.py sdist && pip install dist/xblock-dalite-0.1.tar.gz
 
 test:
+	mkdir -p var
+	rm -r .coverage
```

Comment:
@pomegranited This is the other way around. I have the method under test which has 100% coverage, then I (by mistake) break the tests so coverage drops, but after re-running the tests `coverage report` still stubbornly showed 100% coverage. Only after I removed `.coverage` it showed proper coverage. This might be tested by: reverting this change, running make test, commenting out a bunch of tests, running make test again (should still show 100% coverage). 

I have added `-rf` to the makefile in: 6434eba987e1be31f4305cf6a197f9e6f29501dc

As to 100% coverage --- it's Eugeny who started it :)